test/regress/run_tests.sh

```diff
@@ -25,12 +25,30 @@ run_tests () {
     if test -f /regress/tests/$DIR/regression.diffs; then cat /regress/tests/$DIR/regression.diffs; fi
 }
 
+insert_greeting () {
+    for f in /regress/tests/common/expected/*; do 
+    echo -e "
+\t\tSPQR router admin console
+\tHere you can configure your routing rules
+------------------------------------------------
+\tYou can find documentation here 
+https://github.com/pg-sharding/spqr/tree/master/docs
+" > tmpfile
+        cat $f >> tmpfile
+        mv tmpfile $f
+    done
+}
+
 
 run_tests "console" "regress_router" "7432"
 run_tests "router" "regress_router" "6432"
 run_tests "pooler" "regress_pooler" "6432"
 run_tests "coordinator" "regress_coordinator" "7002"
 
+run_tests "common" "regress_coordinator" "7002"
```

Comment:
Let's make a comment here? Smth like: these tests are to compare the results of the local and qdb coordinator.